pi/main.py

```diff
@@ -69,17 +71,15 @@ def __init__(self, num,
 # ---- SETTINGS ----
 VERSION = "1.2.0"
 
-PORT = "/dev/serial0"       # Serial Port
-BAUD_RATE = 115200          # Serial baud rate
-
-DEFAULT_BOOT_TIME = 20000   # The estimated time to boot and run the beginnings of the script, in MS. Will be used only if RTC is not live
+DEFAULT_BOOT_TIME = 35000   # The estimated time to boot and run the beginnings of the script, in MS. Will be used only if RTC is not live
 
 GPIO_MODE = GPIO.BCM
 VALVE_MAIN_PIN = 27         # Parker 11/25/26 Main Valve control pin
 VALVE_BLEED_PIN = 22        # ASCO Bleed Valve control pin
 VALVE_1_PIN = 10            # First tank control pin
 VALVE_2_PIN = 9             # Second tank control pin
 VALVE_3_PIN = 11            # Third tank control pin
+GSWITCH_PIN = 23            # G-switch input pin
 
 # Setup our Colleciton objects. Numbers from SampleTiming.xlsx in the drive. All durations are going to be the minimum actuation time
 collection_1 = Collection(1, 40305, 290000, 100, 100, 1,   1270.44, 998.20, False)
```

Comment:
Check that collection_1 upwards_bleed should be set to false and the other collections upwards_bleed to true